libraries/AP_HAL_ChibiOS/hwdef/MicoAir743v2/README.md

```diff
@@ -0,0 +1,96 @@
+# MicoAir743v2 Flight Controller
+
+The MicoAir743v2 is a flight controller designed and produced by [MicoAir Tech](http://micoair.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - BMI088/BMI270 dual IMUs

```

Comment:
```suggestion
 - BMI088/BMI270 dual IMUs
 - Integrated BlueTooth module for telemetry
```
is this standard BT or just BLE which is not compatible with any ArduPilot GCS?

Reply:
It is standard BT, and we usually use the QGC Android APK to check the status and messages for convenience.